RPIPs/RPIP-53.md

```diff
@@ -0,0 +1,192 @@
+
+Houston (or more specifically RPIP-31) introduces a new feature, the **RPL withdrawal address**, that changes those fundamentals.
+
+In Atlas, Rocket Pool sends all of the node's rewards (both from the Beacon Chain during a `minipool distribute` and from the Merkle rewards system via `node claim-rewards`) to the node's withdrawal address. In Houston, this is having its name changed to **primary withdrawal address**. The new RPL withdrawal address is a supplemental, opt-in setting that allows a node to specify a "secondary" withdrawal address. When set, the intent is that the *primary* withdrawal address will receive all of the node's ETH rewards, and the *RPL* withdrawal address will receive all of the RPL rewards.
+
+While making sense on paper, the implementation has an unfortunate consequence. If you're a node operator and you have an RPL withdrawal address set, [you can no longer claim rewards](https://github.com/rocket-pool/rocketpool/blob/a747457fc610aa889af0a13cff5d3d00955525a5/contracts/contract/rewards/RocketMerkleDistributorMainnet.sol#L85). This doesn't affect Beacon Chain earnings, but does impact both RPL rewards and Smoothing Pool ETH rewards. Claiming them *must* be done from the RPL withdrawal address. Whether intended or not, the RPL withdrawal address isn't just an RPL withdrawal address in Houston; it's also a "claimer address" that can exclusively claim rewards intervals from the Merkle distributor contract.
+
+This is problematic for a few reasons.
+- First, it means the RPL withdrawal address has dominion over claiming Smoothing Pool ETH that it won't ever recieve. The node (or more accurately the node's primary withdrawal address), which *will* receive the ETH, can't trigger the claim.
+- Second, the node is still going to be listed in the rewards file, and thus the rewards for the corresponding interval are still going to show up as a "claimable interval" in the Smart Node despite the user not being able to claim from it, and not being able to actually *receive* any of the rewards from that interval. This can obviously be worked around in the Smart Node but it's exactly that: a workaround. The file itself is still going to have the node operator listed, so anyone (such as a third party) that wants to use it to view or claim rewards has to know about and implement this workaround as well.
+
+To make it clear, under the Houston contracts here is the set of rules for claiming rewards:
+
+
+### Vanilla Houston - Rewards for Interval X
+
+*Unless otherwise specified, withdrawal addresses are assessed at the time of the claim, not the time of Interval X.*
+
+| Primary Withdrawal Address | RPL Withdrawal Address Set? | Who Can Claim?   | Who Gets ETH? | Who Gets RPL? |
```

Comment:
Can we word "Who Can Claim?" differently? Usually if an entity claims something, it's for themselves. Maybe "Who Can Initiate Claims?" or "Who Can Send Claim Transaction?"?

Reply:
I changed it to "Who Can Trigger Claims?" since that's how the process is referred to in the rest of the proposal.